server/controllers/column.js

```diff
@@ -0,0 +1,53 @@
+const asyncHandler = require("express-async-handler");
+const Column = require("../models/Column");
+
+exports.getColumns = asyncHandler(async (req, res) => {
+  const columns = await Column.find();
+
+  return res.json(Columns);
+});
+
+exports.getColumn = asyncHandler(async (req, res) => {
+  const column = await Column.findById(req.params.id);
+
+  if (!column) res.status(404).json({ error: "Column not found" });
+
+  return res.status(200).json(column);
+});
+
+exports.createColumn = asyncHandler(async (req, res) => {
+  const { name } = req.body;
+  const newColumn = await Column.create({
+    name: name,
+  });
+
+  return res.status(200).json({ Column: newColumn });
```

Comment:
Also, we aren't adding the column to the board. In order to do this we need to either pass the boardID in the response and add the column id to the board. It's more efficient to package the board with column ids that we can populate when we make the request to the board. As opposed to having to query the columns after the board request.

Reply:
This section is being refactored; Columns need to exist on a board and that means we'll need to actually fetch a `board` through `boardId` before creating a column as we need to attach it to the board itself!